app/models/donation_chart.rb

```diff
@@ -7,9 +7,10 @@ def self.donation_by_date(candidate)
    end
    donation_date = donation_date.to_s.gsub('"', '')
  end
- 
+
  def self.donations_by_issue(candidate)
-   donations_by_issue = candidate.candidate_issues.joins(:donations).group(:issue_id).sum(:amount)
-   donations_by_issue.map { |issue_id, amt| [ Issue.find(issue_id).topic, amt ] }
+   donations_by_issue = candidate.candidate_issues.joins(:donations).
+                                                   group(:issue_id).sum(:amount)
+   donations_by_issue.map {|issue_id, amt| [ Issue.find(issue_id).topic, amt ]}
```

Comment:
Space between { and | missing.<br>Space missing inside }.<br>Space inside square brackets detected.